lib/modules/notifications/AndroidNotifications.js

```diff
@@ -91,4 +91,32 @@ export default class AndroidNotifications {
     }
     return Promise.resolve();
   }
+
+  deleteChannelGroup(groupId: string): Promise<void> {
+    if (Platform.OS === 'android') {
+      if (!(groupId instanceof string)) {
+        throw new Error(
+          `AndroidNotifications:deleteChannelGroup expects an 'string' but got type ${typeof groupId}`
+        );
+      }
+      return getNativeModule(this._notifications).deleteChannelGroup(
+        groupId
+      );
+    }
+    return Promise.resolve();
+  }
+
+  deleteChannel(channelId: string): Promise<void> {
+    if (Platform.OS === 'android') {
+      if (!(channnelId instanceof string)) {
```

Comment:
Shouldn't this be `if (typeof channnelId !== 'string') {`, instanceof string would throw `string undefined`?